server/command.go

```diff
@@ -4,9 +4,9 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"html/template"
```

Comment:
So about that, given that what we return is "Markdown" it makes sense to use `html/template` to make sure the data is properly escaped

Reply:
That is great. I was looking for an answer about escapinh